Annotation/declarative_golden_set.json

```diff
@@ -844,7 +905,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that you are a specialist in a particular field",
                 "embedded clauses": []

```

Comment:
add EC "realise how to accomplish something especially well"

Reply:
Sure, as long as we're not restricting each entry to only one sentence